app/controllers/concerns/listings_toolkit.rb

```diff
@@ -190,8 +171,6 @@ def at_least_one_param_present?
   end
 
   def bumped_in_last_24_hrs?
-    return unless (last_bumped_at = @listing.bumped_at)
-
-    last_bumped_at > 24.hours.ago
+    @listing.bumped_at&.after?(24.hours.ago)
```

Comment:
The lonely operator (`&.`) takes care of the early return and lately we prefer `before?` and `after?` for date/time comparisons.